http/src/request/channel/webhook/update_webhook_message.rs

```diff
@@ -157,6 +163,28 @@ impl<'a> UpdateWebhookMessage<'a> {
         self
     }
 
+    /// Specify an attachment to keep.
+    ///
+    /// If called, all unspecified attachments will be removed from the message.
+    /// If not called, all attachments will be kept.
+    pub fn attachment(mut self, attachment: Attachment) -> Self {
+        self.fields.attachments.push(attachment);
+
+        self
+    }
+
+    /// Specify multiple attachments to keep.
+    ///
+    /// If called, all unspecified attachments will be removed from the message.
+    /// If not called, all attachments will be kept.
+    pub fn attachments(mut self, attachments: impl IntoIterator<Item = Attachment>) -> Self {
```

Comment:
Should both `attachment{,s}` and `file{s,}` exist on this request? What's the difference with the file methods, and when do you use either? This should be documented.